packages/spanner/src/table.js

```diff
@@ -347,10 +349,10 @@ Table.prototype.insert = function(keyVals, callback) {
  *     be yielded. If using a composite key, provide an array within this array.
  *     See the example below.
  * @param {string=} query.index - The name of an index on the table.
+ * @param {number} query.limit - The number of rows to return.
```

Comment:
No, we have two. We can change this if you like though. Essentially the query object is specific to the query, options (which is optional) is defining single use transaction options (min timestamps, etc.)